docs/content/en/release-notes/_index.md

```diff
@@ -235,6 +87,40 @@ To manually restart Stargate nodes:
     kubectl scale deployment <stargate-deployment> --replicas 1
     ```
 
+## K8ssandra 1.4.0 revisions
+
+Release date: 15-November-2021
+
+The following sections summarize and link to key revisions in K8ssandra 1.4.0. For the latest, refer to the [CHANGELOG](https://github.com/k8ssandra/k8ssandra/blob/main/CHANGELOG-1.4.md).
+
+### Changes
+
+* Update to Reaper 3.0.0.
+* Update to Stargate 1.0.40 [#1118](https://github.com/k8ssandra/k8ssandra/issues/1118).
+* Update to cass-operator 1.8.0-rc.1 [#950](https://github.com/k8ssandra/k8ssandra/issues/950).
```

Comment:
As mentioned above, this isn't available yet, but we'll want to update this to 1.9.0.
We can also mention the upgrade to kube-prometheus-stack here if you want.

Reply:
Updated to anticipate 1.9.0 - thanks.